src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Syntax/GreenNode.cs

```diff
@@ -240,6 +248,15 @@ public override string ToString()
         return builder.ToString();
     }
 
+    internal string ToStringInCurrentCulture()
```

Comment:
Right abover here is where ToString instead specifies InvariantCulture.

If these were to match, the ToStringInCurrentCulture change could be moved to ToString.